src/neuroconv/datainterfaces/__init__.py

```diff
@@ -132,3 +132,22 @@
     CsvTimeIntervalsInterface,
     ExcelTimeIntervalsInterface,
 ]
+
+interface_lookup = {interface.__name__: interface for interface in interface_list}
```

Comment:
@garrettmflynn Or would you prefer this to be called just `interfaces`?

Reply:
If we aren't using this internally, it might be justified to keep this responsibility with the end-user.

Reply:
You might just want to disregard this comment. My thinking is not quite pythonic—so I think I'm way off the mark here

Reply:
It's mostly that I look at a line like https://github.com/catalystneuro/nwb-guide/commit/0f17ef6f75fec9549be4af2c8b58c7498cfe47c1#diff-bfe22e422f7a3669b04d7ddfc8b73491eda63907f0a4da680bb6494f98f15d69R11 and see it as being overly complex for downstream usage

Happy to make easier internal references for anything like that - the job of NeuroConv is to make everyone's lives easier, after all

Reply:
Now that I understand what's actually happening, I think `interfaces` is the way to go